tools/get-scala-commit-date

```diff
@@ -10,8 +10,13 @@
 
 [[ $# -eq 0 ]] || cd "$1"
 
-lastcommitdate=$(git log --format="%ci" HEAD | head -n 1 | cut -d ' ' -f 1)
-lastcommithours=$(git log --format="%ci" HEAD | head -n 1 | cut -d ' ' -f 2)
+if git rev-parse --is-inside-work-tree > /dev/null 2>&1; then
+	lastcommitdate=$(git log --format="%ci" HEAD | head -n 1 | cut -d ' ' -f 1)
```

Comment:
Restrict to one log message in the first step?

```
20:38 $ git log -1 --format="%ci"|cut -d' ' -f1
2016-09-15
20:38 $ git log -1 --format="%ci"|cut -d' ' -f2
14:07:08
```

I'm in British Summer Time TZ so I have a TZ part. Not sure if this matters for the purposes of this change,

```
20:38 $ git log -1 --format="%ci"|cut -d' ' -f2-3
14:07:08 +0100
```